src/api/axiosClient.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const axiosClient = axios.create();
 
-axiosClient.defaults.baseURL = 'https://gethired-enterprise.herokuapp.com';
+axiosClient.defaults.baseURL = 'https://gethiredplatzi.herokuapp.com/api/v1';
```

Comment:
Could be a better option put the baseURL into the .env file

Reply:
If it is correct 😀, leave it for now for testing, and when Zajith uploads the changes from his branch it is changed to .env so as not to leave the URIs in the code 😀

Reply:
Yep, now it uses an enviroment variable